internal/runners/packages/import.go

```diff
@@ -147,20 +168,34 @@ func fetchImportChangeset(cp ChangesetProvider, file string, lang string) (model
 	return changeset, err
 }
 
-func commitChangeset(project *project.Project, msg string, changeset model.Changeset) (strfmt.UUID, error) {
-	localCommitID, err := commitmediator.Get(project)
-	if err != nil {
-		return "", errs.Wrap(err, "Unable to get local commit")
-	}
-	commitID, err := model.CommitChangeset(localCommitID, msg, changeset)
-	if err != nil {
-		return "", errs.AddTips(locale.WrapError(err, "err_packages_removed"),
-			locale.T("commit_failed_push_tip"),
-			locale.T("commit_failed_pull_tip"))
```

Comment:
Please also remove this from en-us.yaml.

Reply:
Thanks for catching these!